resources/zib/zib-CareTeam.xml

```diff
@@ -0,0 +1,172 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-careteam-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/CareTeam-v1.0(2020EN)" />
+    <name value="zib CareTeam-v1.0(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-healthprofessional-v3.5-2020EN" />
+    <uri value="https://zibs.nl/wiki/HealthProfessional-v3.5(2020EN)" />
+    <name value="zib HealthProfessional-v3.5(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="CareTeam" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/CareTeam" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CareTeam">
+      <path value="CareTeam" />
+      <short value="CareTeam" />
+      <alias value="ZorgTeam" />
+      <mapping>
+        <identity value="zib-careteam-v1.0-2020EN" />
+        <map value="NL-CM:17.3.1" />
+        <comment value="CareTeam" />
+      </mapping>
+    </element>
+    <element id="CareTeam.name">
+      <path value="CareTeam.name" />
+      <short value="CareTeamName" />
+      <definition value="The name of the care team." />
+      <alias value="ZorgTeamNaam" />
+      <mapping>
+        <identity value="zib-careteam-v1.0-2020EN" />
+        <map value="NL-CM:17.3.12" />
+        <comment value="CareTeamNaam" />
+      </mapping>
+    </element>
+    <element id="CareTeam.participant">
+      <path value="CareTeam.participant" />
+      <slicing>
+        <discriminator>
+          <type value="profile" />
+          <path value="member.resolve()" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CareTeam.participant:contactperson">
+      <path value="CareTeam.participant" />
+      <sliceName value="contactperson" />
+    </element>
+    <element id="CareTeam.participant:contactperson.member">
+      <path value="CareTeam.participant.member" />
+      <short value="CareTeamMember" />
+      <definition value="A contact person of the patient as member of the care team." />
+      <alias value="Contactpersoon" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactPerson" />
+      </type>
+      <mapping>
+        <identity value="zib-careteam-v1.0-2020EN" />
+        <map value="NL-CM:17.3.8" />
+        <comment value="CareTeamMember" />
+      </mapping>
+    </element>
+    <element id="CareTeam.participant:patient">
+      <path value="CareTeam.participant" />
+      <sliceName value="patient" />
+    </element>
+    <element id="CareTeam.participant:patient.member">
+      <path value="CareTeam.participant.member" />
+      <short value="CareTeamMember" />
+      <definition value="The patient as member of the care_team." />

```

Comment:
'care_team' should be replaced by 'care team' (error in the zib itself).

Reply:
Fixed!